bot/cogs/match.py

```diff
@@ -73,16 +77,24 @@ def _picker_embed(self, title):
         embed.insert_field_at(1, name='__Players Left__', value=users_left_str)
         return embed
 
-    def _pick_player(self, picker, pickee):
+   def _pick_player(self, picker, pickee):
         """ Process a team captain's player pick. """
         if any(team == [] for team in self.teams) and picker in self.users:
             picking_team = self.teams[self.teams.index([])]  # Get the first empty team
             self.users_left.remove(picker)
             picking_team.append(picker)
         elif picker == self.teams[0][0]:
-            picking_team = self.teams[0]
+            self.captain_one = picker
```

Comment:
Perhaps remove this? It only gets used in one other place and the captain doesn't need to be set each time they make a pick. You can reference each teams captain using `self.teams[x][0]` like how I did.
```suggestion
```